site/source/docs/porting/Debugging.rst

```diff
@@ -250,7 +250,50 @@ If your code hits an infinite loop, one easy way to find the problem code is to
 
 .. note:: The :ref:`emscripten-runtime-environment-main-loop` may need to be re-coded if your application uses an infinite main loop.
 
+.. _debugging-profiling:
 
+Profiling
+=========
+
+Speed
+-----
+
+To profile your code for speed, build with ``--profiling``, then run the code in
```

Comment:
Is this option documented elsewhere? Could we add a link?

Reply:
Yes, the `emcc --help` page has info. I added a link now.